SETUP.md

```diff
@@ -336,3 +337,17 @@ docker run -p 8888:8888 -d recommenders:cpu
 ```
 
 You can then open the Jupyter notebook server at http://localhost:8888
+
+## Setup guide for making a release
+
+The process of making a new release and publish it to pypi is as follows:
+
+1. Make sure that the code in main passes all the tests (unit and nightly tests).
+2. Update the version number in [reco_utils.py/__init__.py](reco_utils.py/__init__.py)
+3. Create a tag with the version number: `git tag -a 0.6.0 -m "Recommenders 0.6.0"`.
+4. Push the tag to the remote server: `git push origin 0.6.0`.
+5. When the new tag is pushed, a release pipeline is executed. This pipeline runs all the tests again (unit, smoke and integration), 
+generates a wheel and upload it to a [GitHub draft release](https://github.com/microsoft/recommenders/releases).
+6. Fill up the draft release with all the recent changes in the code.
+7. Download the wheel locally, this is a wheel that shouldn't have any bug, since it passed all the tests.
+8. Publish the wheel to pypi: `twine upload ms-recommenders*.whl`
```

Comment:
@anargyri uploading the wheel is enough, or do we also need to upload the tar.gz?

Reply:
It's recommended to upload both. See https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives

Reply:
Apparently in some cases pip fails to install the binary and the tarball is a backup option.

Reply:
I think it's a bit funny to have the process for PyPI publication in this place. The audience here is anyone who wants to use the repo. The publication process is addressed only to the maintainers of the repo.

Reply:
I updated this so both files are uploaded, please @anargyri review again

Reply:
There is an error, "ms-recommenders*" does not cover both files, you need "ms_recommenders*" too.

Reply:
> I think it's a bit funny to have the process for PyPI publication in this place. The audience here is anyone who wants to use the repo. The publication process is addressed only to the maintainers of the repo. It could be in a separate markdown file.

Also I suggest you move the PyPI part to CONTRIBUTING.md. It's not really related to setup of the repo.